x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/components/icon_panel.tsx

```diff
@@ -13,17 +13,10 @@ import type { UsePackageIconType } from '../../../../../hooks';
 import { usePackageIconType } from '../../../../../hooks';
 import { Loading } from '../../../../../components';
 
-const PanelWrapper = styled.div`
```

Comment:
Fix for responsive layout (icon overlap) 2. Integration layout broken 480px, 768px breakpoints
See this comment https://github.com/elastic/observability-design/issues/80#issuecomment-916880833